Messaging.md

```diff
@@ -359,35 +362,77 @@ Note that `outcome_i` is the outcome value itself and not its hash that will be
    * [`int32`:`precision`]
    * [`u16`:`nb_digits`]
 
+### The `oracle_event_timestamp` Type
+
+This type contains timestamp information about when an `oracle_event` is to happen.
+
+#### Fixed `oracle_event_timestamp`
+
+1. type: 55348 (`oracle_event_timestamp_fixed`)
+2. data:
+   * [`u32`:`expected_time_epoch`]
+
+#### Range `oracle_event_timestamp`
+
+1. type: 55350 (`oracle_event_timestamp_range`)
+2. data:
+   * [`u32`:`earliest_expected_time_epoch`]
+   * [`u32`:`latest_expected_time_epoch`]
```

Comment:
Would be cool to add a rational, sum up of the ml discussions here :)

https://mailmanlists.org/pipermail/dlc-dev/2021-February/000015.html